contributing.md

```diff
@@ -48,69 +40,136 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ```
 
-## Before you jump in
+</details>
 
-Is there something that you think should be in Backpack, but currently isn't? Before you dive in, please note that not everything that is a UI component belongs in Backpack.
+### Decisions
 
-> Is this really going to be used by multiple products?
+Conventions and squad decisions are kept in the [decisions folder](/decisions). We recommend familiarising yourself with these.
 
-If you use it multiple times yourself, it might as well be a component, but maybe Backpack is the wrong place for it and it should instead stay in your own codebase. If in doubt, don’t hesitate to get in touch with us!
+### Installing Node
 
-> I’m sure this should be in Backpack!
+Backpack is developed using Node, using LTS for the Node version and `^5.6.0` for npm.
```

Comment:
Node LTS is 8.11.3 now! Also can we link to https://github.com/Skyscanner/ensure-node-env?

Reply:
Will do!